frontend/src/FindReplaceView.svelte

```diff
@@ -118,7 +118,7 @@
           if (result === "modified") {
             for (const resource of results[result]) {
               resourceNodeDataMap[resource["id"]] = {
-                modified: true,
+                lastModified: true,
               };
```

Comment:
Same suggestion as the other changes – probably better to not overwrite the entire object.